application/common/util-native/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+import tools.forma.android.config.NdkAbi
+import tools.forma.android.config.CMake
+
+androidNative(
```

Comment:
I don't know deeper behavior of `androidNative`, but I think it will become "god target" same as our `androidLibrary`. In that way, will be better to restrict behavior by making real `nativeUtil`.

Reply:
There are no external dependencies allowed for now, so I guess it's not a god target

Reply:
Cpp modules have to apply `com.android.library` plugin and at this moment you have not choice to opt out of this unless you're adding native code like a binary artifact(e.g. '*.so'). The point, that we could call it `nativeUtil` but there always will be 'android' inside...

Reply:
Ok, let it remain as it is